packages/playground/windows/playground/App.cpp

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.#include "pch.h"
+
+#include "ReactPackageProvider.h"
+
+using namespace winrt::playground;
+using namespace winrt::playground::implementation;
+
+using namespace winrt;
+using namespace Windows::ApplicationModel::Activation;
+using namespace Windows::Foundation;
+using namespace Windows::UI;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::Storage;
+using namespace Windows::Storage::Streams;
+using namespace Windows::Graphics::Imaging;
+using namespace Windows::Storage::Pickers;
+
+struct App : ApplicationT<App> {
+  void OnLaunched(LaunchActivatedEventArgs const &) {
+    const hstring xamlString =
```

Comment:
Why is this not defined in a regular xaml file?

Reply:
If this is unavoidable (not sure why it would be), you could also use a multiline string literal (C++ 11 raw string) to make this a bit less annoying to edit.